docs/user/debugging/android.md

```diff
@@ -36,6 +36,9 @@ adb shell am start -n org.mozilla.samples.glean/mozilla.telemetry.glean.debug.Gl
 
 The `logPings` command doesn't trigger ping submission and you won't see any output until a ping has been submitted. You can use the `sendPings` command to force a ping to be sent, but it could be more desirable to trigger the pings submission on their normal schedule. For instance, the `baseline` and `events` pings can be triggered by moving the app out of the foreground and the `metrics` ping can be triggered normally if it is overdue for the current calendar day. See the [ping documentation](../pings/index.md) for more information on ping scheduling to learn when pings are sent.
 
+Options that are set using the `adb` flags are not immediately reset and will
+persist until the application is closed or manually reset.
+
```

Comment:
One other caveat that was on the dtmo article that I wasn't sure whether it was still valid:

```md
**Note**: Glean will always attempt to collect data for the ping that was
requested using the `sendPing` command line switch.
However, if no data is recorded by the application, nothing will be sent.
The `baseline` ping is _guaranteed_ to always be sent, since it’s populated
by Glean itself.
```

Can someone confirm this was still the case? If so we should add something about this.

Reply:
This is still the case, unless the ping has `send_if_empty` to True (see [here](https://mozilla.github.io/glean/book/user/pings/custom.html#defining-a-custom-ping)).  It would be good to add this information here.